src/topology/algebra/infinite_sum.lean

```diff
@@ -607,6 +607,17 @@ lemma set.finite.summable_compl_iff {s : set β} (hs : s.finite) :
   summable (f ∘ coe : sᶜ → α) ↔ summable f :=
 (hs.summable f).summable_compl_iff
 
+lemma has_sum_ite_eq_extract [decidable_eq β] (hf : has_sum f a) (b : β) :
+  has_sum (λ n, ite (n = b) 0 (f n)) (a - f b) :=
+begin
+  convert has_sum_subtype_iff_indicator.mp (({b} : finset β).has_sum_iff_compl.mp hf),
```

Comment:
In the origin PR I suggested a proof that notes that `has_sum (function.update f b 0)` is `has_sum (λ n, ite (n = b) 0 (f n))`, and uses `has_sum.update`. Does that proof work here?

Reply:
Probably (it'd be the same typeclass assumptions), but it doesn't seem like an obvious improvement?